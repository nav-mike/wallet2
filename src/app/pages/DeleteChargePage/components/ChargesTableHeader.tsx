import React from 'react';

export function ChargesTableHeader() {
  return (
    <tr>
      <th>Date</th>
      <th>Value</th>
      <th>Type</th>
      <th></th>
    </tr>
  );
}
