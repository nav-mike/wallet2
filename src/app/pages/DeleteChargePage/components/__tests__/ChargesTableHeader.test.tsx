import * as React from 'react';
import { render } from '@testing-library/react';
import { ChargesTableHeader } from '../ChargesTableHeader';

describe('<ChargesTableHeader />', () => {
  it('should render and match the snapshot', () => {
    const header = render(
      <table>
        <thead>
          <ChargesTableHeader />
        </thead>
      </table>,
    );
    expect(header.container.firstChild).toMatchSnapshot();
  });
});
