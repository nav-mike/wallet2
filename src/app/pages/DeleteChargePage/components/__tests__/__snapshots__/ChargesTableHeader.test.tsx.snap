// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChargesTableHeader /> should render and match the snapshot 1`] = `
<table>
  <thead>
    <tr>
      <th>
        Date
      </th>
      <th>
        Value
      </th>
      <th>
        Type
      </th>
      <th />
    </tr>
  </thead>
</table>
`;
